src/DUWOagent/DUWOagent.mas2g

```diff
@@ -1,5 +1,5 @@
 use "../../dependencies/contextvh-1.0.1-jar-with-dependencies.jar" as environment 
-	with project = "vhproject_duplicate", stakeholders = ["DUWO"].
+	with project = "vhproject_highbud", stakeholders = ["DUWO"].
```

Comment:
at the moment when i created the branch duplicate didn't have enough budget to be properly tested. I'll change it